test/extensions/filters/http/grpc_json_transcoder/grpc_json_transcoder_integration_test.cc

```diff
@@ -880,5 +889,125 @@ TEST_P(GrpcJsonTranscoderIntegrationTest, UTF8) {
       false);
 }
 
+TEST_P(GrpcJsonTranscoderIntegrationTest, DisableStrictRequestValidation) {
+  HttpIntegrationTest::initialize();
+
+  // Transcoding does not occur from a request with the gRPC content type.
+  // The contents in the body do not matter, they are passed through to upstream.
+  testTranscoding<bookstore::GetShelfRequest, bookstore::Shelf>(
```

Comment:
could you add comment on how no-transcoding is verified in the test.

Reply:
Done. Yes, both are verified, added comments.